### PR TITLE
profiles/monitor: init

### PR DESCRIPTION
**Pull request by zo-el**

### CODE REVIEW

**Comment by samrose** [comment]
I am downloading and trying this out now @zo-el

File: modules/services/holo-monitor.nix [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+      partOf = [ "holo-monitor.service" ];
+      timerConfig.OnCalendar = "*:0/05";
+    };
+    services.holo-monitor = {
+        after = [ "network.target" "holochain-conductor.service" ];
+        path = [ config.services.holochain-conductor.package ];
+        wantedBy = [ "multi-user.target" ];
+
+        serviceConfig = {
+          ExecStart = "${pkgs.nodejs-12_x}/bin/node ${pkgs.holo-monitor}/src/main.js";
```
Comment (samrose) [inline modules/services/holo-monitor.nix]: We are going to add EnvironmentFile here per https://www.freedesktop.org/software/systemd/man/systemd.exec.html#EnvironmentFile=

File: .gitignore [hunk 2]
```diff
@@ -1,2 +1,3 @@
 *.qcow2
 /result*
+.env
```
Comment (yegortimoshenko) [inline .gitignore]: What's that for?

File: modules/services/holo-monitor.nix [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+  };
+
+  config = mkIf cfg.enable {
+  systemd = {
+    timers.holo-monitor = {
+      wantedBy = [ "timers.target" ];
+      partOf = [ "holo-monitor.service" ];
+      timerConfig.OnCalendar = "*:0/10";
+    };
+    services.holo-monitor = {
```
Comment (yegortimoshenko) [inline modules/services/holo-monitor.nix]: Missing newline here.

File: modules/services/holo-monitor.nix [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+        after = [ "network.target" "holochain-conductor.service" ];
+        path = [ config.services.holochain-conductor.package ];
+        wantedBy = [ "multi-user.target" ];
+
+        serviceConfig = {
+          EnvironmentFile="./.env";
+          ExecStart = "${pkgs.nodejs-12_x}/bin/node ${pkgs.holo-monitor}/src/main.js";
+          KillMode = "process";
```
Comment (yegortimoshenko) [inline modules/services/holo-monitor.nix]:
```suggestion
```

File: modules/services/holo-monitor.nix [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+          Restart = "always";
```
Comment (yegortimoshenko) [inline modules/services/holo-monitor.nix]:
```suggestion
```

File: modules/services/holo-monitor.nix [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+    enable = mkEnableOption "Holo Monitor";
+
+    package = mkOption {
+      default = pkgs.holo-monitor;
+      type = types.package;
+    };
+  };
+
+  config = mkIf cfg.enable {
+  systemd = {
```
Comment (yegortimoshenko) [inline modules/services/holo-monitor.nix]: Incorrect indentation here.

File: profiles/holoportos/holo-monitor/default.nix [hunk 7]
```diff
@@ -0,0 +1,74 @@
+{ config, lib, pkgs, ... }:
```
Comment (yegortimoshenko) [inline profiles/holoportos/holo-monitor/default.nix]: `holo-monitor` should not be a sub-profile of HoloPortOS.

**Comment by samrose** [comment]
What is the status of this now?

**Comment by zo-el** [comment]
This is put on hold for now since we are not doing hosting for release.

**Comment by yegortimoshenko** [comment]
This heavily conflicts with current `develop`. Let's re-submit this after shipment.

**Pull request closed without merging**